pkg/task/backup.go

```diff
@@ -35,6 +35,7 @@ const (
 	flagGCTTL = "gcttl"
 
 	defaultBackupConcurrency = 4
+	defaultBackupMaxConcurrency = 128
```

Comment:
```suggestion
	maxBackupConcurrency = 128
```

128 sounds too small, as 128-core CPU is quite feasible.